.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
```

Comment:
Any cool configs available that we should be using?

Reply:
Reading the [documentation](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md) it looks to me that most of the configurable options are very tailored per project. Which packages to include and which are dependent on which (linked). Whether packages should be restricted or public.

the only one I can think of, would be the the [changelog](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md) configuration. 

Are we going to use changesets ourselves when updates are made to this repo? Then we could introduce a script that automatically resets the package versions and bins all changelog files whenever someone starts a new project using this template.